pom.xml

```diff
@@ -88,6 +88,12 @@
             <organization>Hazelcast</organization>
             <organizationUrl>https://hazelcast.com</organizationUrl>
         </developer>
+        <developer>
+            <id>frant-hartm</id>
```

Comment:
Should this be the integration team? I don’t know if any other projects are a sole developer.